bjoern/py2py3.h

```diff
@@ -30,6 +31,18 @@
 #define _Bytes_Resize(obj, len) _PyString_Resize(obj, len)
 #define _FromLong(n) PyInt_FromLong(n)
 #define _Unicode_EncodeLatin1(u) (Py_INCREF(u),u)
+
+#ifdef __GNUC__
+static PyObject *_Unicode_Concat(PyObject *l, PyObject *r) __attribute__ ((unused));
```

Comment:
What does this declaration do, why is it necessary?

Reply:
It is against warrnings, if function is not used, but as this is not a generic header, this can be dropped.

Reply:
`python setup.py build` started to report
```gcc
...
gcc -pthread -fno-strict-aliasing -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -DNDEBUG -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -fPIC -DWANT_SENDFILE=1 -DWANT_SIGINT_HANDLING=1 -Ihttp-parser -I/usr/include/libev -I/usr/include/python2.7 -c bjoern/common.c -o build/temp.linux-x86_64-2.7/bjoern/common.o -std=c99 -fno-strict-aliasing -fcommon -fPIC -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -g
In file included from bjoern/common.c:2:
bjoern/py2py3.h:35:18: warning: ‘_Unicode_Concat’ defined but not used [-Wunused-function]
 static PyObject *_Unicode_Concat(PyObject *l, PyObject *r) {
                  ^~~~~~~~~~~~~~~
...
```

should i return this definition?